src/ui/inspector/PlotElementsPool.vue

```diff
@@ -0,0 +1,333 @@
+                    :element-object="element"
+                    :parent-object="parentObject"
+                    :allow-drop="allowDrop"
+                    @dragstart-custom="moveFrom($event, Y_AXIS_3)"
+                    @drop-custom="moveTo($event, index, Y_AXIS_3)"
+                />
+                <li
+                    v-if="yAxis3.length > 0"
+                    class="js-last-place"
+                    @drop="moveTo($event, yAxis3.length, Y_AXIS_3)"
+                ></li>
+            </element-item-group>
+        </ul>
+        <div
+            v-if="yAxis1.length === 0
+                && yAxis2.length === 0
+                && yAxis3.length === 0"
+        >
+            No contained elements
+        </div>
+    </div>
+</div>
+</template>
+
+<script>
+import _ from 'lodash';
+import Search from '../components/search.vue';
+import ElementItem from './ElementItem.vue';
+import ElementItemGroup from './ElementItemGroup.vue';
+
+const Y_AXIS_1 = 1;
+const Y_AXIS_2 = 2;
+const Y_AXIS_3 = 3;
+
+export default {
+    components: {
+        Search,
+        ElementItemGroup,
+        ElementItem
+    },
+    inject: ['openmct'],
+    data() {
+        return {
+            yAxis1: [],
+            yAxis2: [],
+            yAxis3: [],
+            isEditing: this.openmct.editor.isEditing(),
+            parentObject: undefined,
+            currentSearch: '',
+            selection: [],
+            contextClickTracker: {},
+            allowDrop: false
+        };
+    },
+    created() {
+        this.Y_AXIS_1 = Y_AXIS_1;
+        this.Y_AXIS_2 = Y_AXIS_2;
+        this.Y_AXIS_3 = Y_AXIS_3;
+    },
+    mounted() {
+        const selection = this.openmct.selection.get();
+        if (selection && selection.length > 0) {
+            this.showSelection(selection);
+        }
+
+        this.openmct.selection.on('change', this.showSelection);
+        this.openmct.editor.on('isEditing', this.setEditState);
+    },
+    destroyed() {
+        this.openmct.editor.off('isEditing', this.setEditState);
+        this.openmct.selection.off('change', this.showSelection);
+
+        if (this.compositionUnlistener) {
+            this.compositionUnlistener();
+        }
+    },
+    methods: {
+        setEditState(isEditing) {
+            this.isEditing = isEditing;
+            this.showSelection(this.openmct.selection.get());
+        },
+        showSelection(selection) {
+            if (_.isEqual(this.selection, selection)) {
+                return;
+            }
+
+            this.selection = selection;
+            this.elementsCache = {};
+            this.listeners = [];
+            this.parentObject = selection && selection[0] && selection[0][0].context.item;
+
+            if (this.compositionUnlistener) {
+                this.compositionUnlistener();
+            }
+
+            if (this.parentObject) {
+                this.composition = this.openmct.composition.get(this.parentObject);
```

Comment:
Once we have the parentObject, you can get the yAxisIds and elements for each yAxis from the configuration of the parent object.